lang/py/src/avro/datafile.py

```diff
@@ -331,6 +341,19 @@ def _read_block_header(self):
       uncompressed = snappy.decompress(data)
       self._datum_decoder = io.BinaryDecoder(StringIO(uncompressed))
       self.raw_decoder.check_crc32(uncompressed);
+    elif self.codec == 'zstandard':
+      length = self.raw_decoder.read_long()
+      data = self.raw_decoder.read(length)
+      uncompressed = bytearray()
+      dctx = zstd.ZstdDecompressor()
+      with dctx.stream_reader(StringIO(data)) as reader:
+        while True:
+          chunk = reader.read(16384)
+          if not chunk:
+            break
+          else:
+            uncompressed.extend(chunk)
```

Comment:
The else seems superfluous here. Consider

```python
if not chunk:
    break
uncompressed.extend(chunk)
```

Reply:
Thanks! Updated the PR.